<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterInput</class>
 <widget class="QWidget" name="ParameterInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParameterInput</string>
  </property>
  <widget class="QPushButton" name="pushButton_Prev">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>450</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&lt; Prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Next">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>450</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>451</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="14" column="0" rowspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Chessboard Point Size</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_ChessboardPointSizeDescription">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ChessboardCellSize">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Cell size (mm)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Car width                            (m)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_FileName">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>23</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Text file path</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_Load">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" rowspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Instrinsic Parameter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_CarWidth">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Wheel to wheel (m)</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_CameraToBumper">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Cam to bumper (m)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Camera Height                   (m)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Chessboard Cell Size       (mm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_CarHeight">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Bottom to top (m)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Camera to bumper             (m)</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>CameraNumber                  (m)</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_CameraNumber">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">default(0)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="3">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" rowspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Camera Install Position    (mm)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_CameraInstallDescription">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_CameraInstallPosition">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Position (mm)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="3">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="3">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Chessboard Distance         (m)</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLineEdit" name="lineEdit_CarToChessboard_1">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Cam to target_1(m)</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QLineEdit" name="lineEdit_CarToChessboard_2">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Cam to target_2(m)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="lineEdit_ChessboardSizeX">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">X</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QLineEdit" name="lineEdit_ChessboardSizeY">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string notr="true">Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>9</y>
     <width>951</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 235, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>450</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>269</y>
     <width>450</width>
     <height>173</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>pushButton_Prev</zorder>
  <zorder>pushButton_Next</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>image_1</zorder>
  <zorder>image_2</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_CarWidth</tabstop>
  <tabstop>lineEdit_CarHeight</tabstop>
  <tabstop>lineEdit_CameraToBumper</tabstop>
  <tabstop>lineEdit_ChessboardCellSize</tabstop>
  <tabstop>lineEdit_ChessboardSizeX</tabstop>
  <tabstop>lineEdit_ChessboardSizeY</tabstop>
  <tabstop>lineEdit_CarToChessboard_1</tabstop>
  <tabstop>lineEdit_CarToChessboard_2</tabstop>
  <tabstop>lineEdit_CameraNumber</tabstop>
  <tabstop>pushButton_Prev</tabstop>
  <tabstop>pushButton_Next</tabstop>
  <tabstop>lineEdit_CameraInstallPosition</tabstop>
  <tabstop>pushButton_Load</tabstop>
  <tabstop>pushButton_ChessboardPointSizeDescription</tabstop>
  <tabstop>lineEdit_FileName</tabstop>
  <tabstop>pushButton_CameraInstallDescription</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
